engine/transaction/draft/create_op_return_draft_test.go

```diff
@@ -0,0 +1,142 @@
+package draft_test
+
+import (
+	"context"
+	"encoding/hex"
+	"strings"
+	"testing"
+
+	sdk "github.com/bitcoin-sv/go-sdk/transaction"
+	"github.com/bitcoin-sv/spv-wallet/engine/transaction"
+	"github.com/bitcoin-sv/spv-wallet/engine/transaction/draft"
+	"github.com/bitcoin-sv/spv-wallet/engine/transaction/draft/outputs"
+	"github.com/bitcoin-sv/spv-wallet/models/request/opreturn"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateOpReturnDraft(t *testing.T) {
+	const maxDataSize = 0xFFFFFFFF
```

Comment:
WDYT about putting this value a bit more "globally" potentially in `.../outputs/op_return.go`?

Reply:
It is a BSV related test case, and it is checked by sdk, 
I will comment this and change name, that this is a maximum data that can be pushed with OP_PUSH. But I will leave it here as otherwise it will be part of production code that is not used in production.

Reply:
Ok